ogc-rs/src/audio.rs

```diff
@@ -1,203 +0,0 @@
-        }
-    }
-
-    /// Get the count of bytes, left to play, from the audio DMA interface.
-    fn get_dma_bytes_left() -> u32 {
-        unsafe { ogc_sys::AUDIO_GetDMABytesLeft() }
-    }
-
-    /// Get the audio DMA flag.
-    fn get_dma_enable_flag() -> u16 {
-        unsafe { ogc_sys::AUDIO_GetDMAEnableFlag() }
-    }
-
-    /// Get the DMA transfer length configured in the audio DMA interface.
-    fn get_dma_length() -> u32 {
-        unsafe { ogc_sys::AUDIO_GetDMALength() }
-    }
-
-    /// Get the main memory address for the DMA operation.
-    fn get_dma_address() -> u32 {
-        unsafe { ogc_sys::AUDIO_GetDMAStartAddr() }
-    }
-
-    /// Reset the stream sample count register.
-    fn reset_sample_count() {
-        unsafe {
-            ogc_sys::AUDIO_ResetStreamSampleCnt();
-        }
-    }
-
-    /// Set the sample count for the stream trigger.
-    fn set_trigger_count(count: u32) {
-        unsafe {
-            ogc_sys::AUDIO_SetStreamTrigger(count);
-        }
-    }
-
-    /// Get streaming sample rate.
-    fn get_samplerate() -> SampleRate {
-        let r = unsafe { ogc_sys::AUDIO_GetStreamSampleRate() };
-        SampleRate::try_from(r).unwrap()
-    }
-
-    /// Get the sampling rate for the DSP interface.
-    fn get_dsp_samplerate() -> SampleRate {
-        let r = unsafe { ogc_sys::AUDIO_GetDSPSampleRate() };
-        SampleRate::try_from(r).unwrap()
-    }
-
-    /// Set the sample rate for the streaming audio interface.
-    fn set_samplerate(samplerate: SampleRate) {
-        unsafe {
-            ogc_sys::AUDIO_SetStreamSampleRate(samplerate.into());
-        }
-    }
-
-    /// Set the sampling rate for the DSP interface.
-    fn set_dsp_samplerate(samplerate: SampleRate) {
-        // TODO: Check implementation.
-        let sample_rate: u32 = samplerate.into();
-
-        unsafe {
-            ogc_sys::AUDIO_SetDSPSampleRate(sample_rate as u8);
-        }
-    }
-
-    /// Get the play state from the streaming audio interface.
-    fn get_playstate() -> PlayState {
-        let r = unsafe { ogc_sys::AUDIO_GetStreamPlayState() };
-        PlayState::try_from(r).unwrap()
-    }
-
-    /// Set the play state for the streaming audio interface.
-    fn set_playstate(playstate: PlayState) {
-        unsafe {
-            ogc_sys::AUDIO_SetStreamPlayState(playstate.into());
-        }
-    }
-
-    /// Get streaming volume on the left channel.
-    fn get_volume_left() -> u8 {
-        unsafe { ogc_sys::AUDIO_GetStreamVolLeft() }
-    }
-
-    /// Set streaming volume on the left channel.
-    fn set_volume_left(volume: u8) {
-        unsafe { ogc_sys::AUDIO_SetStreamVolLeft(volume) }
-    }
-
-    /// Get streaming volume on the right channel.
-    fn get_volume_right() -> u8 {
-        unsafe { ogc_sys::AUDIO_GetStreamVolRight() }
-    }
-
-    /// Set streaming volume on the right channel.
-    fn set_volume_right(volume: u8) {
-        unsafe { ogc_sys::AUDIO_SetStreamVolRight(volume) }
-    }
-}
```

Comment:
Any reason to why this file was completely deleted?

Reply:
This was a mistake, I wanted to revert some changes from non `pub` to pub. I've added everything back again.